Packs/Microsoft365Defender/Integrations/Microsoft365Defender/Microsoft365Defender.yml

```diff
@@ -82,20 +81,25 @@ configuration:
   required: false
   type: 8
 - display: Application ID (Deprecated)
+  hidden: true
   name: app_id
   required: false
   type: 4
-  hidden: true
 - display: Tenant ID (for Client Credentials mode) (Deprecated)
+  hidden: true
   name: tenant_id
   required: false
   type: 4
-  hidden: true
 - display: Client Secret (for Client Credentials mode) (Deprecated)
+  hidden: true
   name: enc_key
   required: false
   type: 4
-  hidden: true
+- defaultvalue: '1'
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
```

Comment:
Why was this added?

Reply:
For sure it was not modified by me, so don't ask me